docs/source/usage_guides/accelerate_training.mdx

```diff
@@ -77,10 +86,10 @@ The flexibility of PyTorch comes at a price - usually the same pythonic logic is
 This may lead to a situation where it takes longer for CPU to schedule the work on Habana accelerator than it is effectively computed by it.
```

Comment:
This may lead to a situation where it takes longer for CPU to schedule the work on Gaudi than it is effectively computed by it.

Reply:
Done